File: clients/rust/src/models/msg_namespace_get_in.rs
// this file is @generated
use serde::{Deserialize, Serialize};

#[derive(Clone, Debug, Deserialize, Serialize)]
pub struct MsgNamespaceGetIn {
    pub name: String,
}

impl MsgNamespaceGetIn {
    pub fn new(name: String) -> Self {
        Self { name }
    }
}

File: clients/javascript/src/models/msgNamespaceGetIn.ts
// this file is @generated

export interface MsgNamespaceGetIn {
    name: string;
}

export const MsgNamespaceGetInSerializer = {
    // biome-ignore lint/suspicious/noExplicitAny: intentional any
    _fromJsonObject(object: any): MsgNamespaceGetIn {
        return {
            name: object['name'],
        };
    },

    // biome-ignore lint/suspicious/noExplicitAny: intentional any
    _toJsonObject(self: MsgNamespaceGetIn): any {
        return {
            'name': self.name,
        };
    }
}